proto/src/determined/api/v1/api.proto

```diff
@@ -279,6 +279,27 @@ service Determined {
       tags: "Cluster"
     };
   }
+  // Post new maintenance messages.
+  rpc PostMaintenanceMessage(PostMaintenanceMessageRequest)
```

Comment:
The name of `PostMaintenanceMessage` could be misread as describing the HTTP method POST, which can be confusing—especially since it's not an HTTP POST. What do you think of the name `SetMaintenanceMessage`?

Reply:
agree that `SetMaintenanceMessage` sounds better ✔️